spec/requests/api/v1/applications_spec.rb

```diff
@@ -31,14 +31,35 @@ def create_resource(**attrs)
   end
 
   describe("accept") do
-    before { api_post(api_v1_application_accept_path(123)) }
+    before { api_post(accept_api_v1_application_path(123)) }
 
     specify { expect(response).to(be_method_not_allowed) }
   end
 
-  describe("reject") do
-    before { api_post(api_v1_application_reject_path(123)) }
+  describe "POST /api/v1/npq-applications/:ecf_id/reject" do
+    let(:application) { create(:application, lead_provider: current_lead_provider) }
 
-    specify { expect(response).to(be_method_not_allowed) }
+    context "when application is pending" do
```

Comment:
Worth extracting these to a shared context?

Reply:
👍  with following the context pattern we have established now for other services